docs/build/platform-build-guides/building-ios-applications.md

```diff
@@ -255,6 +255,62 @@ end
 ```
 :::
 
+### Xcode 16 Known Issues
+
+Apple introduces big changes with Xcode, which it develops every year. You can find the errors and solutions encountered for the first time after the Xcode 16 transition under this title.
+
+#### Error: `PBXGroup` attempted to initialize an object with unknown ISA `PBXFileSystemSynchronizedRootGroup`
+
+After Xcode 16 migration, when you add a new target to your project, you will get a reference error if this target is not added correctly. This error is caused by Xcode writing the reference differently in the `.pbxproj` file, which must be read to compile the project and where the project references are written.
+
+```
+
+Error: PBXGroup attempted to initialize an object with unknown ISA PBXFileSystemSynchronizedRootGroup from attributes: 
+{
+  "isa"=>"PBXFileSystemSynchronizedRootGroup", "exceptions"=>["5E7769042CAEDA6A0010DF2F"], 
+"explicitFileTypes"=>{}, "explicitFolders"=>[], "path"=>"ImageNotification", "sourceTree"=>"<group>"
+}
+
+```
+
+This error is caused by the target files added to the project being added as a `folder` and not as a `group`. Xcode uses a `group` structure to `group` files together in the project directory and creates its references accordingly. If you have created a `folder` in the project directory in a way other than Xcode, you will encounter the above error during build because this reference will be misspelt.
```

Comment:
```suggestion
This error is caused when target files are added to the project as a `folder` rather than as a `group`. Xcode uses a `group` structure to organize files within the project directory and creates references accordingly. If a `folder` is manually created in the project directory outside of Xcode, a misspelled reference may result, leading to the error during the build process.
```